frame/crowdloan-rewards/Cargo.toml

```diff
@@ -23,7 +23,7 @@ libsecp256k1 = { version = "0.7.0", default-features = false, features = [
     "static-context",
 ] }
 hex-literal = "0.3"
-balances = { package = "pallet-balances", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false }
+balances = { package = "pallet-balances", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, features = [ "std" ]}
```

Comment:
this will break wasm builds

Reply:
I seem to be able to build wasm successfully while including the std feature in Balances. Is there something I'm missing?
Running `cargo build -p picasso-runtime` in both the project root and inside `./frame/crowdload-rewards` builds successfully.

Reply:
Try enabling benchmarking using `--features runtime-benchmarks`